operators/tokenizer/bert_tokenizer.cc

```diff
@@ -407,18 +407,23 @@ void KernelHfBertTokenizer::Compute(const ortc::Tensor<std::string>& input,
 
   std::list<OffsetMappingType> offset_map;
 
-  // Only compute offset mapping if optional output for it exists.
+  // Only compute offset mapping for optional output if it exists.
```

Comment:
nit: i think this comment is confusing - so we're not computing offset mapping _for_ an optional output right - we're computing it if the optional output asking for it exists. perhaps if the original comment is unclear as well, we can change it to something like "Offset mapping is an optional output - we only compute it if the user demands it, i.e. `compute_offset_mapping` is `true`"

Reply:
ah okay got it now. will update the comment here. thanks!